src/client/index.js

```diff
@@ -499,6 +497,10 @@ $('button[type=submit]').click(e => {
         reader.readAsText(file);
     }
     if (filenum == 0) runquery();
+
+    if (window.innerWidth <= 925) {
+        openTab(circuitTabClass);
+    }
```

Comment:
Why do you insist on checking the window size? You can call openTab always. On desktop layout, this does nothing; on mobile, it does the right thing. The check is not needed, and the magic number in the condition introduces a risk of accidentally making a discrepancy between this and CSS later in the development.

Reply:
Right. I overlooked this fact. I just pushed a quick fix to this and removed the redundant if statement.